Cargo.lock

```diff
@@ -9747,7 +9747,7 @@ dependencies = [
  "indoc",
  "libc",
  "memoffset",
- "parking_lot 0.12.1",
+ "parking_lot 0.11.2",
```

Comment:
Is the downgrade here expected?

Reply:
Reverted it. Seems really strange. Each time I run `cargo update sqlx`, it keeps updating this dependency as well, for some reason.